tellerlogger/src/main/java/com/kiliansteenman/teller/logger/data/room/MapConverter.kt

```diff
@@ -0,0 +1,32 @@
+package com.kiliansteenman.teller.logger.data.room
+
+import androidx.room.TypeConverter
+
+internal class MapConverter {
```

Comment:
This converter is fine. The alternative is adding another Entry table or something which only adds complexity.

Reply:
Agreed, other option would be to serialize it to json, but I don't really want to add a dependency on Gson/Moshi/Other json lib just for that.